crates/fuel-core/src/service/sub_services.rs

```diff
@@ -38,6 +38,8 @@ pub type BlockProducerService = fuel_core_producer::block_producer::Producer<
 >;
 pub type GraphQL = crate::fuel_core_graphql_api::service::Service;
 
+const LOG_THRESHOLD_MS: u64 = 2000;
```

Comment:
It would be nice if you made it configurable via CLI arguments=) Also you can use `Duration`

Reply:
No problem. I'll push it up to the CLI level! Should I include a default for the CLI/make the field optional?

Reply:
Yep, let's use 2 seconds as a default value for now=)